### ISSUE TITLE
The chorus creates gating effect

### ISSUE THREAD

**Issue by muziker** [jpcima/string-machine#31]
Using v0.1.0-125-g7050cb1

The chorus on the string seems to create a gating effect, or a clipping effect. The strings by themselves sound ok. If the string machine is fed into a separate chorus with a high enough dry level, it sounds a lot better.

**Comment by jpcima** [comment]
Yes the synth's chorus is known to produce high spikes, which can sometimes clip.
I'm not sure of the best solution to fix this atm.
